RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+View.swift

```diff
@@ -279,9 +283,23 @@ private extension String {
 			}
 		}
 	}
+
+	var asInstantDataField: ArbitraryDataFieldKind? {
+		nilIfEmpty.map {
+			if let timeInterval = Int64($0) {
+				.instant(Date(timeIntervalSince1970: TimeInterval(timeInterval)))
+			} else {
+				.primitive(self)
+			}
+		}
+	}
 }
 
 private extension GatewayAPI.ProgrammaticScryptoSborValue {
+	enum TypeName: String {
+		case instant = "Instant"
```

Comment:
So "Instant" is the only typename we support? For non-instants, is `typeName` simply undefined?

Reply:
`Instant` it is the only one we do identify and act upon currently. For non-instants we do ignore the typeName, even if it is defined.